maintainers/responsibilities.md

```diff
@@ -0,0 +1,48 @@
+# Maintainer responsibilities
+
+## Notifications
+
+A maintainer must watch discussions and proposals from the different
+communication channels: GitHub, Discourse, and Matrix.
+
+To monitor GitHub, you must have your own entry in the `.github/CODEOWNERS`
+file. GitHub will then notify you and request you for review for Pull Requests
+that change files that you own.
+
+Start by choosing a restricted set of files you want to watch. Keep in mind
+that choosing a broad set of files can lead to too frequent notifications.
+Then, ask for `write` access to the repository, which is needed for the
```

Comment:
refine: ask who?

Reply:
If we use the bot, just make a PR to edit the maintainer file. @Mic92 please add a note/link which one exactly.